tests-api/app/use_cases/calculate_score.rb

```diff
@@ -31,23 +31,28 @@ def calculate_results
     @correct_questions = 0
 
     answers.map do |answer|
-      correct = answered_correctly?(
-        answer[:question_uuid],
+      correct_choices = correct_choices(answer[:question_uuid])
+      correct_answer = answered_correctly?(
+        correct_choices,
         answer[:selected_choices],
       )
 
-      @correct_questions += 1 if correct
+      @correct_questions += 1 if correct_answer
 
       {
         question_uuid: answer[:question_uuid],
-        correct: correct,
+        correct_answers: correct_choices,
+        correct: correct_answer,
```

Comment:
> maybe rename correct to avoid confusion with correct_answers ? answered_correctly maybe?

I was thinking about the public API here, to make it more understandable for the outside world. WDYT?

```ruby
{
  score: "1/1",
  results: [
    {
      question_uuid: "uuid",
      correct_answers: ["a", "b", "c"],
      answered_correctly: true, # instead of :correct
    },
  ],
}
```